### PR TITLE
Recent updats

### LINKED ISSUE #22: add a CODEOWNERS file
Requested by @elhmn , add a file listing out the various owners of this project so they can be notified on PR creation. 

- Name it `CODEOWNERS`
- Save it in the `.github` directory.

### PR DESCRIPTION
**Pull request by andrew21-mch**
fixes #22

Changes coming into this repo from this branch include:
 - This is just an update that i got from Elroys Branch
 - 
 -

### CODE REVIEW

File: src/app/bot.py [hunk 1]
```diff
@@ -106,6 +106,35 @@ def menu_actions(update: Update, context: CallbackContext) -> None:
     ))
 
 
+def all_callback(update: Update, message_text: str, reply_markup: InlineKeyboardMarkup) -> None:
```
Comment (elhmn) [inline src/app/bot.py:109]: I am not sure why this function is called `all_callback` when it is not used on all actual callbacks. 
Maybe we should rename it with something else ?
```suggestion
def sendMarkupAnswer(update: Update, message_text: str, reply_markup: InlineKeyboardMarkup) -> None:
```

**Review by elhmn — APPROVED** [review]
This looks good I left a comment few comments

File: src/app/bot.py [hunk 2]
```diff
@@ -123,8 +152,14 @@ def handler():
     dispatcher.add_handler(CommandHandler('help', help_command))
     dispatcher.add_handler(CommandHandler('menu', menu_command))
 
+    # menu test
+    dispatcher.add_handler(CommandHandler('menut', menu_test))
```
Comment (elhmn) [inline src/app/bot.py:156]:
```suggestion
    dispatcher.add_handler(CommandHandler('menu_test', menu_test))
```

Comment (andrew21-mch) [reply src/app/bot.py]: thanks, i just updated that

File: src/app/bot.py (hunk 1 above)
Comment (andrew21-mch) [reply src/app/bot.py]: yea thanks, i just did that

**Pull request merged**